Integrations/integration-RsaNetWitnessPacketsAndLogs.yml

```diff
@@ -98,9 +99,15 @@ script:
         } else if ((typeof data) ==='object') {
             extractFromData(data);
         }
+        //May be empty elements which should be filtred
+        queryResults = queryResults.filter(function (element) {
+            return element != undefined || element != null;
+        });
 
-        if(retVal.md.length > 0) {
-            var hr = tblToMd(command, retVal.md,'', '<br>');
+        if(queryResults.length > 0) {
+            retVal.md = queryResults;
+            retVal.context.NetWitness.Events = queryResults;
```

Comment:
,Search can be any entity If you know how it fits better, then you should write design. And I think this what was decided by me and Lior or Jonathan